var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -52,6 +54,8 @@ class Cube(AutotoolsPackage):
     depends_on('qt@5:', when='@4.3.0: +gui')
     depends_on('qt@4.8:', when='@4.2.0:4.2.999 +gui')
 
+    conflicts('~gui', when='@4.4:')
```

Comment:
Hm, woudn't it then become reasonable to make the `gui` variant `default=True`? Otherwise people would now directly get a conflict on a simple `spack install cube`

Reply:
Good point. My `packages.yaml` enabled this, so I missed it. I have pushed an updated version with `default=True`.

configure complains about missing qmake even if `--without-gui` has been specified: `configure: WARNING: Cannot find qmake in your PATH. Try to specify the exact path using the option --with-qt=<path> or to switch off the compilation of gui using option --without-gui.`

I also could not find another flag to disable it (`--without-cubegui` also did not work etc.).

Reply:
Hm, looks like a configure bug to me. Do you want to ping upstream about this?

Reply:
It's also interesting that we don't take our internal dependencies but only a bundled (dependency-vendored) cube here. Is this intentional?

Reply:
After thinking about this again, I guess it actually does not make a lot of sense to build the bundle without the GUI, since the bundle is just cubelib + cubew + cubegui (and there are already packages for cubelib and cubew). I will rework the package to build versions 4.4+ from just the cubegui tarball and update the PR in a bit.

Reply:
I have just pushed an updated PR that uses the cubegui tarball and the cubelib package when building 4.4+.